templates/validator/dynconf_update.sh.epp

```diff
@@ -51,3 +53,18 @@ for data in blacklists ticklemacros issuer_keyring participants products service
 done
 
 test -x /usr/local/bin/kdk_update && /usr/local/bin/kdk_update
+
+<%if $bobcat::validator::refresh_api { -%>
+
+if [ $? -eq 0 ]; then
+        REFRESH=true
+fi
+
+if [ "$REFRESH" = true ] ; then
+	#let files cool down
+	sleep 5
+	mosquitto_pub -t <%= $bobcat::validator::refresh_api %> -m '{"command":"refresh"}'
```

Comment:
Is `$bobcat::validator::refresh_api` a topic or a boolean?

Reply:
both, common pattern in puppet that only requires one parameter for both configuration and inclusion. There are no conflicts in doing this.